Sessionless Product Detail.md

```diff
@@ -0,0 +1,50 @@
+# Sessionless Product Details
+
+**Introduction & Base Definitions**
+
+
+Authentication protocols (auth protocols) are gatekeepers that verify the identity of a person to give them access to specific programs or data. They include a set of rules or procedures that govern how data is transmitted across electronic devices. The most common uses of auth protocols are passwords, but others include one-time-passwords, biometrics, and magic links. 
+
+- An Application Programming Interface (API) is  a software intermediary that allows two  applications to share data with each other
+- An API Key is  a unique code that identifies an application or user
+- An API Token is a piece of code that contains information about a user and is used to secure access to an API
+- Encryption is the process of turning data into code to prevent unauthorized access
+
+**Problem Statement**
+
+Many auth protocols are client-server, where the client supplies some secret information (usually name and email address) to authenticate requests.
+
+There are multiple problems with this set up:
+- Requires users to enter in personally identifiable information which can be hacked presenting legal risks 
+- Requires encryption to prevent sensitive data leaks using https which slows and limits authentication opportunities (can’t connect to anything other than standard computers)
```

Comment:
This is worded such that it sounds like the data leaks happen because of https. Maybe something like, "Encryption protocols, like https, provide some protection against data leaks, but they limit authentication opportunities (https for example, can't be used to connect embedded devices)

Reply:
Updated with latest merge